src/common/cli_wrapper.cpp

```diff
@@ -128,6 +128,38 @@ void CLIWrapper::parse(int argc, char **argv) {
   }
 }
 
+void CLIWrapper::parseAliases() {
+  if(aliases_.empty())
```

Comment:
aliases are YAML nodes?

Reply:
No, AliasTuple apparently, this deserves a comment, where to find the actual object definition.